From ec345f1f7bc4454dea08a29413c855225ae573b8 Mon Sep 17 00:00:00 2001
From: "A. Maitland Bottoms" <bottoms@debian.org>
Date: Sat, 10 Feb 2018 20:29:24 -0500
Subject: [PATCH 09/27] make acc happy

Header files need to be compilable using abi-compliance-checker.
---
 gnuradio-runtime/include/gnuradio/py_feval.h |  1 +
 gnuradio-runtime/include/gnuradio/sys_pri.h  | 10 +++++-----
 2 files changed, 6 insertions(+), 5 deletions(-)

diff --git a/gnuradio-runtime/include/gnuradio/py_feval.h b/gnuradio-runtime/include/gnuradio/py_feval.h
index cef168c8f0..89491af0b4 100644
--- a/gnuradio-runtime/include/gnuradio/py_feval.h
+++ b/gnuradio-runtime/include/gnuradio/py_feval.h
@@ -23,6 +23,7 @@
 #ifndef INCLUDED_GR_PY_FEVAL_H
 #define INCLUDED_GR_PY_FEVAL_H
 
+#include <Python.h>
 #include <pmt/pmt.h>
 #include <gnuradio/feval.h>
 
diff --git a/gnuradio-runtime/include/gnuradio/sys_pri.h b/gnuradio-runtime/include/gnuradio/sys_pri.h
index adceb91b9d..d251455a11 100644
--- a/gnuradio-runtime/include/gnuradio/sys_pri.h
+++ b/gnuradio-runtime/include/gnuradio/sys_pri.h
@@ -23,7 +23,7 @@
 #define INCLUDED_GNURADIO_SYS_PRI_H
 
 #include <gnuradio/api.h>
-#include <realtime.h>
+#include <gnuradio/realtime.h>
 
 /*
  * A single place to define real-time priorities used by the system itself
@@ -31,10 +31,10 @@
 namespace gr {
 
   struct GR_RUNTIME_API sys_pri {
-    static rt_sched_param python();		  // python code
-    static rt_sched_param normal();		  // normal blocks
-    static rt_sched_param gcell_event_handler();
-    static rt_sched_param usrp2_backend();	  // thread that services the ethernet
+    static struct GR_RUNTIME_API rt_sched_param python();		  // python code
+    static struct GR_RUNTIME_API rt_sched_param normal();		  // normal blocks
+    static struct GR_RUNTIME_API rt_sched_param gcell_event_handler();
+    static struct GR_RUNTIME_API rt_sched_param usrp2_backend();	  // thread that services the ethernet
   };
 
 } /* namespace gr */
-- 
2.11.0

